src/main/resources/templates/fragments/userBar.html

```diff
@@ -9,8 +9,8 @@
     <ul class="mobile-hidden" id="navigation">
       <li>
         <ul class="mobile-hidden"
-            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(user)}">
-          <li class="user" id="signed-in-user" th:text="${user.getId() + ':'}"></li>
+            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
```

Comment:
Is it necessary to have the full class name, rather than the shortened one?

Reply:
Spoken in person, but yes, this needs to be fully type qualified. If we want to change this we can either refactor UserModel or change the default spring type locator in the expression language.
Either choice is more work than we care to do atm.